Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/LoadContentScene.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit;
+#if SCENE_TRANSITIONS_ENABLED
```

Comment:
Move this to the Examples/ExampleHub/Scripts. Atm, it's not very extensible. If you want to call LoadContent again you can't change the parameters

Reply:
Moved it.